### PR TITLE
Cover js binding graph with tests

### PR DESCRIPTION
**Pull request by margaritiko**
Cover js binding graph with [Mocha.js](https://mochajs.org/) tests.

**It is unit testing in the Browser.**

## How to launch these tests?
1. `cd project/src/com/google/daggerqueryui/tests`
2. `npm install mocha chai --save-dev`
3.  click on the `tests_runner.html ` and open the page 

Screenshot 2020-09-15 at 16 25 59

### CODE REVIEW

File: project/src/com/google/daggerqueryui/tests/binding_graph_test.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var assert = chai.assert;
+
+describe('Binding Graph', function() {
+  it('adds one edge correctly', function() {
+    bindingGraph.clear();
```
Comment (o-y) [inline project/src/com/google/daggerqueryui/tests/binding_graph_test.js:19]: This same line is being called in every test. You could consider moving this into a beforeEach hook. https://mochajs.org/#hooks

Comment (margaritiko) [reply project/src/com/google/daggerqueryui/tests/binding_graph_test.js:19]: Thanks, [done](https://github.com/googleinterns/dagger-query/pull/41/commits/9d7ddf8a267f15c62783fca633714e2702be938d) 🍰

File: project/src/com/google/daggerqueryui/tests/binding_graph_test.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+  it('adds one edge correctly', function() {
+    bindingGraph.addEdge('com.google.A', 'com.google.B');
+
+    const expectedNodes = [
+      [0, 'A', 'com.google.A'],
+      [1, 'B', 'com.google.B']
+    ];
+
+    assert.sameDeepMembers(bindingGraph.getNodes().map(node => [node.id, node.label, node.title]), expectedNodes);
```
Comment (kwchan1995) [inline project/src/com/google/daggerqueryui/tests/binding_graph_test.js]: Would it be possible to assert the raw data structure for clarity?

e.g.
```
assert.sameDeepMembers(bindingGraphs.getNodes(), [{id: 0, label: 'A', title: 'com.google.A'}, {id: 1, label: 'B', title: 'com.google.B'}]
```

**Review by kwchan1995 — APPROVED** [review]
Nice!

**Review by Ratchette — APPROVED** [review]

File: project/src/com/google/daggerqueryui/tests/binding_graph_test.js (hunk 2 above)
Comment (margaritiko) [reply project/src/com/google/daggerqueryui/tests/binding_graph_test.js]: Yes, good idea, thanks
[Done](https://github.com/googleinterns/dagger-query/pull/41/commits/7cdde8a0ba085d2fb5711a791d36da3d0faaef41) 🌱

**Pull request merged**